src/lib.rs

```diff
@@ -91,30 +70,30 @@ pub mod actors;
 pub mod clock;
 pub mod fut;
 pub mod io;
-pub mod msgs;
 pub mod registry;
 pub mod sync;
 pub mod utils;
 
-pub use actor::{
+pub use actix_rt::{Arbiter, System, SystemRunner};
```

Comment:
Should we maybe re-export whole `actix_rt` as `rt` module?

It also has useful `spawn` function

Reply:
i dont think we need rt

Reply:
ok